CODE_OF_CONDUCT.md

```diff
@@ -1 +1,7 @@
-all repositories in the [scala](https://github.com/scala) and [scalacenter](https://github.com/scalacenter) organizations are covered by the Scala Code of Conduct: https://scala-lang.org/conduct/
+all repositories in these organizations:
+
+* [scala](https://github.com/scala)
```

Comment:
Perhaps we could use longer words, such as:
```suggestion
* The [Scala Compiler Developers](https://github.com/scala).
* The [Scala Center](https://github.com/scalacenter), the open source foundation supporting, promoting, and coordinating the development and evolution of the Scala language.
* The [Programming Methods Laboratory](https://github.com/lampepf) at the _École Polytechnique Fédérale de Lausanne_ (EPFL).
```

Reply:
I believe that the intention is to have the names be the names of the GitHub "organization", so if anything it would be "The Scala Programming Language". I don't think that's clearer.

Reply:
yeah, idk, mixed feelings. I think I'll merge as-is